src/main/cpp/subsystems/Flywheel.cpp

```diff
@@ -38,7 +38,8 @@ units::radian_t Flywheel::GetAngle() {
 }
 
 units::radians_per_second_t Flywheel::GetAngularVelocity() {
-    return units::radians_per_second_t{m_encoder.GetRate()};
+    return (GetAngle() - m_lastAngle) /
+           units::second_t{(std::chrono::steady_clock::now() - m_lastTime)};
```

Comment:
```suggestion
           units::second_t{std::chrono::steady_clock::now() - m_lastTime};
```

Reply:
If we remove the parentheses then it will divide before we subtract the times.

Reply:
False. The `units::second_t{}` ensures the subtraction happens first because the result is being passed to the `units::second_t` constructor.